frontend/src/pages/FeedPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import '../styles/FeedPage.css';
+
+import { Card, Image, Button, Icon, Divider, FeedExtra } from 'semantic-ui-react'
+import { getProfiles } from "../utils/api";
+
+function FeedPage() {
+
+  const [ profiles, setProfile] = useState([]);
+  useEffect(() => {
+    const demoCall = async () => {
+      const profiles = await getProfiles();
+      console.log(profiles);
+      setProfile(profiles);
+    };
+    demoCall();
+  }, []);
+
+  return (
+    <div class="feedpage">
+      {profiles.map(value =>
+        <Card.Group itemsPerRow={1} style ={{display: 'flex', marginLeft: 'auto', marginRight: 'auto'}}>
```

Comment:
usually classes + css files are preferred over inline styling, but this is fine for product showcase

Reply:
"fine for product showcase" lol